daml-lf/transaction-test-lib/src/main/scala/lf/transaction/test/TransactionBuilder.scala

```diff
@@ -188,6 +188,7 @@ object TransactionBuilder {
       byKey: Boolean = true,
   ): Exercise =
     Exercise(
+      observers = Set.empty,
```

Comment:
We probably have to add the `observers` to the argument of the definition. 
The question is if we should provide a default value. I think `Set.empty[String]` as a default value make sens.

Reply:
I'd prefer to not have a default value. I think they are just too dangerous.

Reply:
Note that this is a library for writing test, and it may used by external code. 
That said, I am OK with no default value. In any case the `observers` should be an argument of the method.